includes/discovery/sensors/fanspeed/xos.inc.php

```diff
@@ -19,10 +19,10 @@
     if (is_numeric($value)) {
         $descr = "Fan Speed $modindex";
         // round function used to round limit values to hundreds to avoid h/w/l limits being changed on every discovery as a change of 1rpm for fan speed would cause the limit values to change since they're dynamically calculated
```

Comment:
This sensor is guessing limits outside the normal function :/
Guessed limits are supposed to only be applied at initial discovery, but this applies them every time.

It would probably be best to just delete this.